src/app/routes/applications/new-role/components/role-type-did/role-type-did.component.spec.ts

```diff
@@ -44,65 +41,28 @@ describe('RoleTypeDidComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should display error message about already existing element ', () => {
-    component.dids = ['did:ethr:0x' + stringWithLength(40)];
-    fixture.detectChanges();
-
-    const { didInput } = selectors(hostDebug);
-    didInput.value = 'did:ethr:0x' + stringWithLength(40);
-    dispatchInputEvent(didInput);
-    fixture.detectChanges();
-
-    const { matError } = selectors(hostDebug);
-    expect(matError.textContent).toContain(
-      'This DID already exist on the list'
-    );
-  });
-
-  it('should display error message when element is not a did', () => {
-    component.dids = ['did:ethr:0x' + stringWithLength(40)];
-    fixture.detectChanges();
-
-    const { didInput } = selectors(hostDebug);
-    didInput.value = 'did:ethr:0x';
-    dispatchInputEvent(didInput);
-    fixture.detectChanges();
-
-    const { matError } = selectors(hostDebug);
-    expect(matError.textContent).toContain('DID format is invalid');
-  });
-
```

Comment:
Were these test cases not longer relevant? They seem like decent test cases to me still

Reply:
Well, the logic has been moved to `select-did` component. There we have input with autocomplete that contains DIDs from the DID book. There is also handling validations etc. However i can move test `should display error message when element is not a did` there. The test `should display error message about already existing element` is already moved.